pkg/kubeapiserver/options/authentication.go

```diff
@@ -163,11 +163,6 @@ func (s *BuiltInAuthenticationOptions) WithWebHook() *BuiltInAuthenticationOptio
 	return s
 }
 
-func (s *BuiltInAuthenticationOptions) Validate() []error {
```

Comment:
I'd rather this go in other direction. Not all flag combinations are safe and `Validate` can check that.  For instance, if you set `--oidc-issuer-url`, you must set `--oidc-client-id`